diff --git a/Cargo.toml b/Cargo.toml
index 3f2ac2e..b65f758 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -46,8 +46,6 @@ cursive = { version = "0.12.0", default-features = false, optional = true, featu
 [target.'cfg(windows)'.dependencies.pancurses]
 version = "0.16.0"
 features = ["win32"]
-[target.'cfg(unix)'.dependencies]
-cursive = { version = "0.12.0", optional = true }
 
 
 [build-dependencies]
